packages/kit/src/packaging/test/errors/no-lib-folder/package.json

```diff
@@ -0,0 +1,5 @@
+{
+	"name": "error-no-lib-folder",
+	"version": "1.0.0",
+	"description": "validate no lib folder error output"
```

Comment:
I think we need to add `private:true` if I read the Discord chat about publishing correctly

Reply:
Oh yeah, added to the remaining fixtures as well